protoc/CppFileGenerator.cpp

```diff
@@ -233,6 +237,19 @@ void FileGenerator::GenerateBuildDescriptors(Printer* printer) {
       "assigndescriptorsname", GlobalAssignDescriptorsName(m_output_name));
 
     printer->Print("}  // namespace\n");
+#else
+    printer->Print(
+      "namespace {\n"
+      "\n"
+      "inline void protobuf_AssignDescriptorsOnce() {\n"
+      "  static ::google::protobuf::internal::once_flag once;\n"
+      "  ::google::protobuf::internal::call_once(once,\n"
```

Comment:
Likewise this once stuff seems to be from v3.7.0:
https://github.com/protocolbuffers/protobuf/commit/0894e07536af88065d462cdc9d8e807c0723ef4d

Reply:
Done.